src/components/inputs/toggle/toggle.js

```diff
@@ -22,6 +22,17 @@ export class Toggle extends React.PureComponent {
     size: 'big',
   };
 
+  emitChange = () => {
+    const event = {
+      target: {
+        id: this.props.id,
+        name: this.props.name,
+        value: !this.props.isChecked,
+      },
+    };
+    this.props.onChange(event);
```

Comment:
Why are we constructing the event ourselves? 🤔

Reply:
I can't get it to work with the native event. You are welcome to try it, but I spent about an hour on it and got nowhere with RTL. The shape is the same now as the native event though.

Reply:
Oh really? Sorry about that...

Hmm that's weird, I would have expected this to work. I can have a look over the weekend.

Reply:
switching to `defaultChecked` fixed the probelm